src/data.js

```diff
@@ -78,6 +78,25 @@ const pages = [
       'Blogger',
     ],
   },
+  {
+    name: 'Conrad Mugisha',
+    description:
+      'Software Developer. Lover of challenges',
+    url: 'https://aless.co/uses',
```

Comment:
This is Alessia Bellisario's "uses" page I believe

Reply:
oh snap, forgot to edit that